### PR TITLE
wording refine: 家目录 -> Home 目录

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
wording refine: 家目录 -> Home 目录

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: hybrid-deployment-topology.md [hunk 1]
```diff
@@ -102,4 +102,4 @@ aliases: ['/docs-cn/dev/hybrid-deployment-topology/']
 > - 编辑配置文件模版时，注意修改必要参数、IP、端口及目录。
 > - 各个组件的 deploy_dir，默认会使用 global 中的 `<deploy_dir>/<components_name>-<port>`。例如 tidb 端口指定 4001，则 deploy_dir 默认为 '/tidb-deploy/tidb-4001'。因此，在多实例场景下指定非默认端口时，无需再次指定目录。
 > - 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
-> - 如果部署目录配置为相对路径，会部署在用户家目录下。
+> - 如果部署目录配置为相对路径，会部署在用户的 Home 目录下。
```
Comment (Oreoxmt) [inline hybrid-deployment-topology.md:105]: @TomShawn geo-distributed-deployment-topology.md “家目录”

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: hybrid-deployment-topology.md (hunk 1 above)
Comment (TomShawn) [reply hybrid-deployment-topology.md:105]: done

**Pull request merged**